generator/vogtle_data_generator.py

```diff
@@ -58,49 +58,120 @@ def __init__(self, directory=None):
         if directory:
             self.directory = directory
 
-    def generate_inspections(self, rows):
+    @staticmethod
+    def generate_itaac_effort_str(itaac_id, effort_id,
+                                  effort_type, effort_range):
+        """Generate string, bar seperated value of itaac_effort entry
+
+        :param rows: Number of rows to generate
+        """
+        actual = random.choice(list(effort_range))
+        estimate = random.choice(list(effort_range))
+        itaac_effort_str = ("\n%s|%s|%s|%s|%s" % (effort_id,
+                                                  itaac_id,
+                                                  effort_type,
+                                                  actual,
+                                                  estimate))
+
+        return itaac_effort_str
+
+    def generate_itaac_efforts(self, itaac_ids):
+        """Generate ITAAC Efforts
+
+        :param itaac_ids: IDs of ITAACS to generate efforts for
+        """
+        output = []
+        effort_types = ["ITAAC", "Program", "Reactive/Allegations",
+                        "Technical", "Total"]
+
+        with open('{}itaac_efforts.csv'
+                  .format(self.directory), 'w+') as output_file:
+
+            # write header
+            output_file.write("id|itaac_id|effort_type|actual|estimate")
+            for effort_id in range(1, len(itaac_ids)+1):
+                actual_total = 0
+                estimated_total = 0
+
+                itaac_id = itaac_ids[effort_id - 1]
+                for effort_type in effort_types:
+
+                    if effort_type == 'Total':
+                        actual = actual_total
+                        estimate = estimated_total
+                    else:
+                        actual = random.choice(list(range(0, random.choice(
+                            [40, 60, 80, 100, 120]))))
+                        estimate = random.choice(list(range(0, random.choice(
+                            [40, 60, 80, 100, 120]))))
+
+                        actual_total += actual
+                        estimated_total += estimate
+
+                    output.append("\n%s|%s|%s|%s|%s" % (effort_id,
+                                                        itaac_id,
+                                                        effort_type,
+                                                        actual,
+                                                        estimate))
+
+            output_file.write(''.join(output))
+
+    def generate_inspections(self, rows, generate_efforts_flag=False):
         """
         Generate synthetic data for Inspections
+
+        :param rows: Number of rows to generate
+        :param generate_efforts_flag: Boolean flag to generate effort hours
         """
         header = "id|itaac_status|icn_status|est_completion_date|" \
-                 "effort_required|facility|targeted_flag|target_amt\n"
+                 "date_received|facility|targeted_flag"
```

Comment:
- [x] Convert to multiline for future header expansion.
- [ ] The old string ended with a blank line which was removed.  Do you still want a new line?

```suggestion
        header = (
            'id|'
            'itaac_status|'
            'icn_status|'
            'est_completion_date|'
            'date_received|'
            'facility|'
            'targeted_flag'
        )
```

Reply:
@calderonmluis Did you want a new line on the header?